sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BulkExecutionOptions.java

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.ImplementationBridgeHelpers;
+import com.azure.cosmos.implementation.batch.BatchRequestResponseConstants;
+import com.azure.cosmos.implementation.spark.OperationContextAndListenerTuple;
+import com.azure.cosmos.util.Beta;
+
+import java.time.Duration;
+
+public final class BulkExecutionOptions {
```

Comment:
We should mark the new classes as well with Beta annotation. So that we can merge this PR early, and then we can remove Beta Annotations once we are ready for GA.